### ISSUE TITLE
[BUG] Testground testkit devnet sometimes starts miners on different chains

### ISSUE THREAD

**Issue by nonsense** [filecoin-project/lotus#6127]
**Describe the bug**
The Testground Lotus testkit used in the `lotus-soup` testplan is not starting devnets correctly. Occasionally miners start mining on different chains, resulting in flaky tests. This should be fixed if we want to enable more elaborate Testground Lotus tests, such as deals end-to-end tests or payment channel tests in networks of more than 1 miner.

**Version (run `lotus version`):**
`4688da51`

**To Reproduce**
1. Run the Testground payment channel stress tests (part of CI) with multiple miners, instead of just 1.
2. See error: `failed to sync fork: synced chain forked at genesis, refusing to sync`

<img width="1360" alt="Screenshot 2021-04-28 at 17 04 20" src="https://user-images.githubusercontent.com/50459/116426891-cb2e0c80-a843-11eb-8272-c8c292c8c275.png">

**Expected behavior**
The `lotus-soup` testkit should be able to consistently start devnets with multiple miners (2, 3, 5, etc.)

**Comment by BigLep** [comment]
@jennijuju : I don't think we should say this is in review.  https://github.com/filecoin-project/lotus/pull/6126 is a mitigation so we can get testground running but this issue is about fixing underlying issues.  I would link #6126 to some of the enable testground tests issues and this issue still needs to get prioritized.

@nonsense : let me know if I have any of that wrong.

**Comment by magik6k** [comment]
If those miners have the same power, this might be related to https://github.com/filecoin-project/lotus/issues/2981
